smartmin/models.py

```diff
@@ -303,6 +325,7 @@ def unicode_csv_reader(utf8_data, dialect=csv.excel, **kwargs):
         if import_results is not None:
             import_results['records'] = len(records)
             import_results['errors'] = num_errors
+            import_results['error_description'] = error_description
```

Comment:
`error_messages` maybe ? Wondering also if we should have a limit on the number of error messages we record... users aren't going to want ~10000 error messages if they really mess up a big import... hmm... @nicpottier ?